### PR TITLE
adding home page

### PR DESCRIPTION
**Pull request by AbhiramVAnand**

### CODE REVIEW

File: home.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>PixelVibe</title>
+    <link rel="stylesheet" href="css/home.css" />
+  </head>
+  <body>
+    <div class="container">
+      <nav class="navbar">
```
Comment (BreadGenie) [inline home.html:11]: @abhiram-anand the home page design looks nice but it'd be much nicer if the navbar is raised to the top

**Review by BreadGenie — CHANGES_REQUESTED** [review]

File: css/home.css [hunk 2]
```diff
@@ -7,7 +7,7 @@ body {
   cursor: default;
 }
 .navbar {
-  margin: 4% 6% 5% 7%;
+  margin: 2% 6% 5% 7%;
```
Comment (BreadGenie) [inline css/home.css:10]: idk if it's just on my machine but this is how it looks for me

Screenshot from 2020-12-29 14-37-17

in my opinion it'd look better if the whole navbar is on top

**Review by BreadGenie — CHANGES_REQUESTED** [review]

**Comment by AbhiramVAnand** [comment]
I think it's only on yours.
I got something like this

**Comment by BreadGenie** [comment]
@abhiram-anand can you provide a screenshot?
also @ankitapuri take a look at this PR

**Comment by ankitapuri** [comment]
> @abhiram-anand can you provide a screenshot?
> also @ankitapuri take a look at this PR

Yeah I have seen it.  I also suggest that the navbar should be above . It's coming inline with the project description.

**Comment by AbhiramVAnand** [comment]
@BreadGenie  here's the screenshot
Screenshot_2020-12-27_PixelVibe

**Comment by Harshit850** [comment]
I had a new design idea for this landing page, i would like to show you, can i?

**Comment by ankitapuri** [comment]
> I had a new design idea for this landing page, i would like to show you, can i?

let's discuss in gitter

**Pull request closed without merging**